CHANGELOG.rst

```diff
@@ -15,6 +15,9 @@ Fixed
 
 Added
 ~~~~~
+* Remove `distutils` dependencies across the project. #5992
+  Contributed by @AndroxxTraxxon
```

Comment:
This better fits the `Changed` changelog subcategory, instead of `Added`.
You can create one, like here https://github.com/StackStorm/st2/pull/5992/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR111

Reply:
Fair point. I'll move it over.